lib/modules/manager/maven-wrapper/artifacts.ts

```diff
@@ -9,7 +9,9 @@ import type { ExecOptions, ExtraEnv } from '../../../util/exec/types';
 import { chmodLocalFile, readLocalFile, statLocalFile } from '../../../util/fs';
 import { getRepoStatus } from '../../../util/git';
 import type { StatusResult } from '../../../util/git/types';
+import { find as findHostRules } from '../../../util/host-rules';
```

Comment:
Nitpick but why `findHostRules`, when in fact only one `HostRule` is returned?

Please just do `import * as hostRules from '../../../util/host-rules';` 
and later on `const { username, password } = hostRules.find({`. 

It's only a minor thing but helps to avoid introducing X different naming and usage patterns for the same function throughout the whole code base. The pattern I mentioned is the one that's already established.